sdf.md

```diff
@@ -165,34 +163,33 @@ Properties are used to model elements of state.
 
 - Qualities of odmProperty
 
-| Quality | Type | Optional | Description |
-|---|---|---|---|
-|id| integer | yes | internal unique identifier for the definition |
-|name|string|yes|human readable name|
-|description|string|yes|human readable description|
-|title|string|yes|human readable title to display|
-|optional| boolean|yes|defines whether this element is optional in an implementation|
+| Quality | Type | Required | Description | Default |
+|---|---|---|---|---|
+|id| integer | yes | internal unique identifier for the definition | N/A |
+|name|string|yes|human readable name| N/A |
+|description|string|yes|human readable description| N/A |
+|title|string|yes|human readable title to display| N/A |
 |include|array|yes|reference to definitions to be included|
-|type|object|yes|reference to a definition to be used as a template for a new definition|
-|readOnly|boolean|yes|Only reads are allowed|
-|writeOnly|boolean|yes|Only writes are allowed|
-|observable|boolean|yes| flag to indicate asynchronous notification is available|
-|contentFormat|string|yes|IANA media type string|
-|units|string|yes|[SenML unit][] code|
-|nullable|boolean|yes|indicates a null value is available for this type|
-|encoding|map|yes|applies additional constraints|
-|scaleMinimum|number|yes|lower limit of value in units|
-|scaleMaximum|number|yes|upper limit of value in units|
-|type|string, enum|yes|JSON data type|
-|minimum|number|yes|lower limit of value in the representation format|
-|maximum|number|yes|upper limit of value in the representation format|
-|multipleOf|number|yes|indicates the resolution of the number in representation format|
-|enum|array|yes|enumeration constraint|
-|pattern|string|yes|regular expression to constrain a string pattern|
-|minLength|integer|yes|shortest length string in octets|
-|maxLength|integer|yes|longest length string in octets|
-|default|number, boolean, string|yes|specifies the default value for initialization|
-|const|number, boolean, string|yes|specifies a constant value for a data item or property|
+|type|object|yes|reference to a definition to be used as a template for a new definition| N/A |
```

Comment:
type, readOnly, writeOnly, observable, etc. should probably *not* be required

Reply:
now not required anymore